integrated_channels/canvas/client.py

```diff
@@ -59,19 +79,39 @@ def delete_course_completion(self, user_id, payload):  # pylint: disable=unused-
     def create_content_metadata(self, serialized_data):
         self._create_session()
 
-        url = '{}/api/v1/accounts/{}/courses/'.format(
+        # step 1: create the course
+        url = CanvasAPIClient.course_create_endpoint(
             self.enterprise_configuration.canvas_base_url,
-            self.enterprise_configuration.canvas_account_id,
+            self.enterprise_configuration.canvas_account_id
         )
-        return self._post(url, serialized_data)
+        status_code, response_text = self._post(url, serialized_data)
+
+        # step 2: upload image_url if present
+        try:
```

Comment:
I forget, does the image url get updated in the single update request put? Whereas here I see we're making the second call.

Reply:
line 100 does it

Reply:
the first post simply ignores even if the image_url is there

Reply:
sorry- was unclear. I was referring to when content metadata is updated, did we figure out that the it only requires the one request?

Reply:
oh yea the update endpoint does handle it, TBH I did not test update but I am certain the endpoint (since it's the same I used) will update image when a change is presnet

Reply:
the issue is only with the create endpoint